auth/src/main/java/com/firebase/ui/auth/ui/phone/CountryListAdapter.java

```diff
@@ -92,6 +102,16 @@ public int getPositionForSection(int index) {
 
     @Override
     public int getSectionForPosition(int position) {
+        if (sections == null) {
```

Comment:
Since we were using "fast scrolling" this method had to be implemented to get the scroll bar to work correctly.

Reply:
Yeah won't show up till you start scrolling, that's normal.  There's nothing I can figure out to fix the jumpiness, will maybe look into it again later if people report it.